jest.config.ts

```diff
@@ -5,6 +5,7 @@ const config: Config = {
   collectCoverage: true,
   coverageDirectory: '.coverage',
   coverageProvider: 'v8',
+  coverageReporters: ['json', 'lcov', 'clover'],
```

Comment:
sorry, I had to remove the test coverage from being printed on every test run, it was unusable.

Reply:
Yea, it just takes up a ton of the screen when you're running tests.